RocketChat/Rocket.Chat
# [IMPROVE] Enhance the `changed_room_privacy` system message text

## Proposed changes (including videos or screenshots)


- Update `changed_room_privacy` system message text to "@user changed room to **public**" or "@user changed room to **private**"



## Issue(s)


## Steps to test or reproduce

Create a team, go to **Team's Information > Edit > Private**, toggle this option and save. A system message should be sent in the team's main room.

## Further comments
